src/Common/GUI/Custom_OptionsGUI.m

```diff
@@ -36,63 +37,221 @@
 % End initialization code - DO NOT EDIT
 
 
-% --- Executes just before Custom_OptionsGUI is made visible.
 function OptionsGUI_OpeningFcn(hObject, eventdata, handles, varargin)
-% WAIT IF OUTPUTS
+
+% This function is called each time the Options Panel is opened.
+%
+% OptionsPanel is a non-modal window with the <uipanel29> ID in handles
```

Comment:
Will it always be fixed to the `uipanel29`name? It sounds oddly specific, but I can see it being so, because "GUIs".

Reply:
Yeap, it should be fixed unless we get highly annoyed by it and decide to change it first in the fig. then everywhere else.